core/state_transition.go

```diff
@@ -602,13 +602,13 @@ func (st *stateTransition) execute() (*ExecutionResult, error) {
 		usedMultiGas     = multigas.ZeroGas()
 	)
 
-	// Check clauses 4-5, subtract intrinsic gas if everything is correct
-	gas, err := IntrinsicGas(msg.Data, msg.AccessList, msg.SetCodeAuthorizations, contractCreation, rules.IsHomestead, rules.IsIstanbul, rules.IsShanghai)
```

Comment:
I don't feel at all strongly, but I'd lean against the variable rename in the name of minimizing diff, but defer to others

Reply:
I actually like this rename. It's more diff-lines, but very easy to see you're not doing something weird